site/src/pages/components/invokers.explainer.mdx

```diff
@@ -326,6 +302,302 @@ Invokers, the names and exact semantics may be subject to change:
+may be announced in a screen reader as "File, button, collapsed", while
+`<button aria-expanded=true>File</button>` may be announced in a screen reader
+as `File, button, expanded`. This is helpful information for users who require
+an alternative of the visual confirmation of the popover being rendered on
+screen.
+
+If the popover element is not in the DOM, browsers should ensure the button has
+the `aria-expanded=false` state. Some websites may chose to render such
+elements only when they are open, often referred to as "conditional rendering".
+Browsers should aim to do their best to cater for this, and as such provide the
+state under the assumpsion that a popover element may appear in the DOM on
+press.
+
+</details>
+
+<br/><br/>
+
+If the `commandfor=` element is a `<dialog>` the button will also recieve an
+`aria-haspopup=dialog`.
+
+<details>
+<summary>Why?</summary>
+
+Some Assistive Technologies will announce that the button will open a dialog,
+for example `<button aria-haspopup=dialog>Delete</button>` may be announced in
+a screen reader as "Delete, button, opens dialog". This can be important
+information for a user as it allows them to anticipate a focus change or a
+change in the pages navigable elements which may otherwise not be quite as
+apparent.
+
+However, we may revisit this decision based on more evidence, research and
+discussion. Buttons that open dialogs rarely come with additional visual
+treatment, and so the lack of a visual analog may allude to the presence of
+`aria-haspopup=dialog` being redundant.
+
+This decision _may also_ be revisited in light of any new elements proposals
+that have the role of `menu`, `listbox`, `tree` or `grid`. It may be proposed
+that, when connected to those elements via `commandfor`, buttons will
+implicitly recieve `aria-haspopup` of the matching type.
+
+</details>
+
+<br/><br/>
+
+If `commandfor=` does not point to an element connected to the DOM then this
+attribute must not be present.
+
+Buttons with these popover commands will also have focus restored to the
+button when the popover closes. This will require the popover keeping a
+reference to the opening button so that it can return the focus to that button.
+
+<details>
+<summary>Why?</summary>
+
+It is important for keyboard users (or other users who rely on focus) to not
+have their focus state lost when performing actions on the page. If the page
+shows a new piece of navigable UI, and shifts the user's focus to that UI,
+then it is important to restore focus when that UI closes, so that the user's
+journey is maintained.
+
+</details>
+
+<br/><br/>
+
+<details>
+<summary>Other considerations not explicitly proposed.</summary>
+
+##### aria-pressed
+
+Given elements will have `aria-expanded`, adding `aria-pressed` would be
+confusing or redundant, and as such won't be proposed for these buttons.
+
+##### aria-controls
+
+While `aria-controls` attempts to establish a similar style of relationship to
```

Comment:
this isn't really accurate - re: parent/child relationship.  i'm not sure why this is mentioned at all if it's not being used / was never implemented for popovers?

Reply:
I tried to add these extra bits as somewhat of an FAQ to field questions of "why hasn't this particular property been implemented", to have some written down rationale that people can refer to when those questions arise.

Reply:
ok, that makes sense.  

so then it's more of a way this is described then - since it's not commonly a parent/child relationship - but more of an association between related elements

Reply:
Would it be better to point out instead that `aria-details` has better support, and applying both is redundant?

Reply:
yah that could work.  so something like

aria-controls is not used due to lack of support among screen readers, and even if it was supported it would be redundant to the association already created due to the implicit aria-details relationship already created.